be/src/util/decimal_types.h

```diff
@@ -133,6 +174,8 @@ template <typename T>
 inline constexpr T get_max() {
     if constexpr (std::is_same_v<T, int128_t>) {
         return INT128_MAX;
+    } else if constexpr (std::is_same_v<T, int256_t>) {
+        return INT256_MAX;
```

Comment:
where is INT256_MAX's definition?

Reply:
[here](https://github.com/StarRocks/starrocks/blob/main/be/src/types/int256.h#L572)